### ISSUE TITLE
Possible incorrect match on StringSigils rule

### ISSUE THREAD

**Issue by radar** [rrrene/credo#313]
### Environment

* Credo version (`mix credo -v`): 0.6.0
* Erlang/Elixir version (`elixir -v`): 1.3.3
* Operating system: Mac OS X.

### What were you trying to do?

I was trying to write documentation for a method in my library, [elastic](https://github.com/radar/elastic). If you checkout to radar/elastic@397b130d8b70e9fd2ace1dd539b400c2020df6b3 and use Credo 0.6.0 you will see this issue happen.

The documentation I have there is:

```elixir
 @doc """
  Makes a request using the PUT HTTP method:

      Elastic.HTTP.put("/answers/answer/1", body: %{
        text: "I like using Elastic Search"
      })

  """
  def put(url, options \\ []) do
    request(:put, url, options)
  end
```

### Expected outcome

Credo should not output a message when the quotes occur inside a `@doc` block.

### Actual outcome

Credo outputs a message:

```
┃ [R] ↓ More than 3 quotes found inside string literal, consider using a sigil
┃       instead.
┃       lib/elastic/http.ex:46 (Elastic.HTTP)
```

I think this may be related to https://github.com/rrrene/credo/issues/248.

**Comment by joshprice** [comment]
Hi Ryan, this is a duplicate of #305 which was fixed in #307

**Comment by rrrene** [comment]
@radar I published `v0.6.1` just now. Could you check if this resolves the issue?

@joshprice `v0.6.1` contains your fix. Thx again!

**Comment by rrrene** [comment]
@radar I will close this assuming it is solved for you. If this is not the case, please re-open!
